src/components/GenreCard/GenreCard.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+    Button
+} from "@mui/material"
+
+type GenreCardProps = {
+    imgSrc: string
+    genreName: string
+    noOfQuizzesAvailable: number
+}
```

Comment:
Suggestion: You can make one folder which contains all your your types.

Reply:
Thanks for suggestion, will make this change before merging 😄